libs/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart.component.ts

```diff
@@ -40,12 +40,26 @@ export class FhirChartComponent implements OnInit {
 
   @Input() emptyMessage: string = 'No data';
 
-  constructor(private readonly configService: FhirChartConfigurationService, public layerManager: DataLayerManagerService) {}
+  constructor(
+    private readonly configService: FhirChartConfigurationService,
+    public layerManager: DataLayerManagerService,
+  ) {}
 
   ngOnInit(): void {
     Chart.register(scaleStackDividerPlugin, annotationPlugin, zoomPlugin);
-
-    // To responsively resize the chart based on its container size, we must set maintainAspectRatio = false
+    document.addEventListener('keydown', (event) => {
+      const chart = Chart.getChart('baseChart');
+      if (chart) {
+        switch (event.key) {
+          case 'ArrowRight':
+            chart.pan({ x: -50 });
+            break;
+          case 'ArrowLeft':
+            chart.pan({ x: 50 });
+            break;
+        }
+      }
```

Comment:
Please make sure the new events don't trigger while a form field is focused (e.g. enrollment form or prediabetes risk factors)

Reply:
you can also test this with the date input in timeline-range-selector

Reply:
> Please make sure the new events don't trigger while a form field is focused (e.g. enrollment form or prediabetes risk factors)

After the update, I tested connecting to Sapphire and launching prediabetes. Now it's working properly when form is in focus.